### PR TITLE
Theme design

### PR DESCRIPTION
**Pull request by itsaruon**
Observer design pattern for the theme functionality

### CODE REVIEW

File: app/src/main/java/com/example/giftly/CThemeScreen.java [hunk 1]
```diff
@@ -15,88 +15,100 @@
...
             case android.R.id.home:
                 this.finish();
                 return true;
         }
         return super.onOptionsItemSelected(item);
-    }
-}
\ No newline at end of file
+}
+*/
```
Comment (plazluis) [inline app/src/main/java/com/example/giftly/CThemeScreen.java]: Maybe delete the commented back button configuration if its not going to be used.

File: app/src/main/java/com/example/giftly/CThemeScreen.java [hunk 2]
```diff
@@ -15,88 +15,100 @@
...
+        public void onThemeChanged(int newColor) {
+            getWindow().getDecorView().setBackgroundColor(newColor);
+        }
+}
+
+
+    //Back button
+    //ActionBar actionBar = getSupportActionBar();
+    //actionBar.setDisplayHomeAsUpEnabled(true);
+
```
Comment (plazluis) [inline app/src/main/java/com/example/giftly/CThemeScreen.java]: Delete the ActionBar methods since the configuration will either be used or deleted.

**Review by plazluis — COMMENTED** [review]
Minor fixes on commented code

File: app/src/main/java/com/example/giftly/CThemeScreen.java (hunk 2 above)
Comment (itsaruon) [reply app/src/main/java/com/example/giftly/CThemeScreen.java]: Removed, thank you

**Review by plazluis — DISMISSED** [review]

**Review by plazluis — APPROVED** [review]

**Review by Mattmcgoo — APPROVED** [review]

**Pull request merged**